packages/query/src/graphql/graphql.module.ts

```diff
@@ -133,9 +132,12 @@ export class GraphqlModule implements OnModuleInit, OnModuleDestroy {
       const pluginHook = makePluginHook([PgPubSub]);
       // Must be called manually to init PgPubSub since we're using Apollo Server and not postgraphile
       options = pluginHook('postgraphile:options', options, {pgPool: this.pgPool});
-      options.replaceAllPlugins.push(PgSubscriptionPlugin);
+      options.replaceAllPlugins = options.replaceAllPlugins || [];
```

Comment:
A newish feature is [??=](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment)

```suggestion
      options.replaceAllPlugins ??= [];
```